src/inmanta/server/agentmanager.py

```diff
@@ -1331,6 +1341,10 @@ async def _make_agent_config(
 password = {opt.influxdb_password.get()}
 interval = {opt.influxdb_interval.get()}
 tags = {config_map_to_str(opt.influxdb_tags.get())}
+
+[server]
+# Required to resolve the IP address to connect back to the Inmanta server
+bind-address={opt.server_bind_address.get()}
```

Comment:
why is that? We have `serveradress` above as well? I would expect the scheduler to use only that.

Reply:
You are right. This should not be there.